### DISCUSSION TITLE
Error Unable to import 'fastapi' en Visual Code

### DISCUSSION THREAD

**Discussion by warbar13** [fastapi/fastapi#10602] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from fastapi import FastAPI
from typing import Union

app = FastAPI()


@app.get("/")
def read_root():
    return {"Hello": "World"}


@app.get("/items/{item_id}")
def read_item(item_id: int, q: Union[str, None] = None):
    return {"item_id": item_id, "q": q}
```

### Description

estoy en modo aprendizaje,y realice la instalacion de fastapi y uvicorn y tengo python version 3.9.1  pero sale que no se ha podido resolver la importacion 
el archivo en VC colocoque main.py  hasta  cambie el nombre de la carpeta por proyecto-env
from fastapi import FastAPI

app = FastAPI()

que sera?  gracias

### Operating System

Windows

### FastAPI Version

este comando me genera error modulo

### Pydantic Version

no tengo instalado

### Python Version

3.9.1

### Additional Context

estoy  desde consola de windows las  instalaciones y el interprete en visual  stude code

gracias  por la asesoria

**Comment by jw-00000** [comment, marked as answer]
If you installed FastAPI, this is probably because you're not using the right Python environment in VS Code. Check [this tutorial by Microsoft](https://code.visualstudio.com/docs/python/tutorial-fastapi), Section "Locally in VS Code", to set up FastAPI in VS Code. Make sure to follow all steps exactly, especially those related to the environment!

**Comment by punkrockcode** [comment]
I had this issue. I was creating the virtual enviroment in a subfolder in the workspace. set venv on workspace folder.
